examples/eager_mlp/mlp_eager_simple.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2023 Nod Labs, Inc
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import logging
+import unittest
+
+import torch
```

Comment:
Can you add this here: https://github.com/nod-ai/SHARK-Turbine/blob/main/tests/examples/aot_mlp_test.py (so we have some test coverage of this example)?